pkg/reconciler/v1alpha1/taskrun/resources/pod.go

```diff
@@ -229,7 +244,29 @@ func MakePod(taskRun *v1alpha1.TaskRun, taskSpec v1alpha1.TaskSpec, kubeclient k
 
 	nopContainer := &corev1.Container{Name: "nop", Image: *nopImage, Command: []string{"/ko-app/nop"}}
 	entrypoint.RedirectStep(cache, len(podContainers), nopContainer, kubeclient, taskRun, logger)
-	podContainers = append(podContainers, *nopContainer)
+	mergedStep, err := mergeStepWithBaseContainer(baseContainer, nopContainer)
```

Comment:
hm, instead of calling `mergeStepWithBaseContainer` several times, is there a way we can call it with all the steps together once we have them? (maybe on `podContainers` below?)

Reply:
Yeah, can do